dolphinscheduler-task-plugin/dolphinscheduler-task-remoteshell/src/main/java/org/apache/dolphinscheduler/plugin/task/remoteshell/RemoteShellTask.java

```diff
@@ -90,7 +90,6 @@ public void init() {
             taskId = TASK_ID_PREFIX + taskExecutionContext.getTaskInstanceId();
         }
         setAppIds(taskId);
-        taskExecutionContext.setAppIds(taskId);
```

Comment:
I don't think it's appropriate to delete the appId directly here. I suggest modifying the logic of ProcessUtils. Currently, it seems that the appId is considered to be only yarn app id, but in fact there are other external application ids, such as remote shell.

cc @Radeity 

https://github.com/apache/dolphinscheduler/blob/738da1cda3731543cdec61c249a051fbc99da929/dolphinscheduler-task-plugin/dolphinscheduler-task-api/src/main/java/org/apache/dolphinscheduler/plugin/task/api/utils/ProcessUtils.java#L187-L193